pkg/api/routes.go

```diff
@@ -99,6 +99,8 @@ func (rh *RouteHandler) SetupRoutes() {
 			rh.DeleteBlobUpload).Methods("DELETE")
 		prefixedRouter.HandleFunc("/_catalog",
 			rh.ListRepositories).Methods(allowedMethods("GET")...)
+		prefixedRouter.HandleFunc("/_oci/ext/discover",
```

Comment:
Make /_oci ... etc a constant
Event _catalog above
under constants/extensions.go

// Reserved
_oci
_catalog
// our stuff
_search etc

Reply:
On top of these routes table, add a comment with this URL

https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints